client/containers/Pub/PubHeader/TitleGroup.tsx

```diff
@@ -4,29 +4,29 @@ import { PubByline, DialogLauncher, PubAttributionDialog } from 'components';
 import { usePageContext } from 'utils/hooks';
 import { getPubPublishedDate } from 'utils/pub/pubDates';
 import { formatDate } from 'utils/dates';
-import { Pub, Release } from 'types';
+import { PatchFn, Pub, PubPageData } from 'types';
 
 import BylineEditButton from './BylineEditButton';
 import EditableHeaderText from './EditableHeaderText';
+import { usePubContext } from '../pubHooks';
 
 type Props = {
-	pubData: Pub & {
-		releases: Release[];
-		isRelease?: boolean;
-	};
-	updatePubData: (...args: any[]) => any;
+	pubData: PubPageData;
+	updatePubData: PatchFn<Pub>;
 };
 
 const TitleGroup = (props: Props) => {
 	const { pubData, updatePubData } = props;
 	const { title, description, isRelease } = pubData;
 	const { communityData, scopeData } = usePageContext();
+	const { submissionState } = usePubContext();
+	const isUnsubmitted = submissionState?.submission.status === 'incomplete';
 	const { canManage } = scopeData.activePermissions;
-	const canModify = canManage && !isRelease;
+	const canModify = canManage && !isRelease && !isUnsubmitted;
```

Comment:
This makes for the peculiar experience of the pub title + byline suddenly becoming editable from within the preview view. A UX question, I don't have strong sense of what to do about it.